### PR TITLE
Milestone 4 : Desktop version

### PR DESCRIPTION
**Pull request by AHP15**
In this pull request I added:
- Styles for the desktop version of the project using media queries
- Images for backgrounds
- Responsive design

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Abdessittir, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] The `ul-items` class that you use in your work cards and technologies cards is not perfectly centered in the card due to a `padding-right` and a default `padding` of the ul element. Kindly try removing the padding of this element so the elements in the cards are all centered.
-Your design:
image
image

-Figma template:
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -155,8 +161,9 @@
     justify-content: center;
     flex-wrap: wrap;
     gap: 5px;
-    padding-right: 15px;
+    width: 100%;
     margin-top: 10px;
+    margin-left: -20px;
```
Comment (GiuseppeTG) [inline style.css]: - [ ] It seems you were not able to center them doing this. The problem is that you have padding messing with this 😅 

.ul-items {
    display: flex;
    align-items: center;
    justify-content: center;
    flex-wrap: wrap;
    gap: 5px;
    width: 100%;
    margin-top: 10px;
    padding: 0 10px;
  }

Kindly remove margin-left: -20px (Is not good practice to use negative margins for this purpose)
Please overwrite the existing padding (the second value of padding should be adjusted as in Figma )

I think this should work just fine 🤓

File: style.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```
Comment (GiuseppeTG) [inline style.css]: [OPTIONAL] I think your font-weight is not working properly. Make sure that when you import from google fonts you actually select every font weight you are going to use. 

This is an example of some font with weights that I have:

Captura de Pantalla 2022-09-28 a la(s) 7 52 13 a  m

Hope this helps 🤓

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @Abdessittir 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Highlights
✅ No linter errors
✅ Descriptive commit messages
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -301,3 +313,97 @@
...
+    width: 920px;
+  }
+
+  .work-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 474px);
+    column-gap: 15px;
+  }
+
+  .about-container {
```
Comment (muneebulrehman) [inline style.css:361]: Optional.
Consider using the same font color as in the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — APPROVED** [review]
Hi @Abdessittir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**